packages/synapse-react-client/src/components/OrientationBanner/OrientationBanner.tsx

```diff
@@ -54,9 +56,10 @@ function OrientationBanner(props: OrientationBannerProps) {
     localStorage.getItem(storageBannerId) === null,
   )
 
-  const theme = useTheme()
-  const defaultSx = {
-    display: { xs: 'none', md: 'unset' },
+  const defaultSx: SxProps<Theme> = theme => ({
+    [theme.breakpoints.down('md')]: {
+      display: 'none',
+    },
```

Comment:
More concise/conventional way to just change the breakpoint styles is to do this. Now we aren't touching md and up.